pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -477,6 +480,18 @@ type SnapshotScheduleSpec struct {
 	StartTime string `json:"startTime,omitempty"`
 }
 
+// QuotaSpec represents the spec for quotas in a pool
+type QuotaSpec struct {
+	// Enabled represents whether pool quotas are set
+	Enabled bool `json:"enabled,omitempty"`
+
+	// MaxBytes represents the quota in bytes
+	MaxBytes uint64 `json:"maxBytes,omitempty"`
+
+	// MaxObjects represents the quota in objects
+	MaxObjects uint64 `json:"maxObjects,omitempty"`
```

Comment:
Referencing https://github.com/rook/rook/pull/7264/files#r578815432

```suggestion
	// MaxBytes represents the quota in bytes
	MaxBytes *uint64 `json:"maxBytes,omitempty"`

	// MaxObjects represents the quota in objects
	MaxObjects *uint64 `json:"maxObjects,omitempty"`
```

Alternately, we could leave `QuotaSpec` as a non-pointer but have the parameters themselves control Rook's implicit behavior.

This will have no effect.
```yaml
quotas: {}
```

This will tell Rook to unset all quotas:
```yaml
quotas:
   maxBytes: 0
   maxObjects: 0
```

This will have rook unset the maxBytes quota but not change the maxObject quota.
```yaml
quotas:
   maxBytes: 0
```

Reply:
This sounds good. Let me test it out.

Reply:
Great. Thanks for contributing and being open to feedback. 🎉

Reply:
Changes made. All tests successful. This definitely seems like to best solution, thanks for the suggestion.